modules/platforms/dotnet/Apache.Ignite.Core/Binary/BinaryBasicNameMapper.cs

```diff
@@ -58,7 +73,12 @@ public string GetTypeName(string name)
             {
                 // Generics are rare, use simpler logic for the common case.
                 var res = IsSimpleName ? parsedName.GetName() : parsedName.GetNameWithNamespace();
-                
+
+                if (!IsSimpleName && NamespaceToLower)
+                {
+                    res = DoForceJavaNamingConventions(res, JavaDomain == null ? null : JavaDomain + '.');
```

Comment:
I don't think we need to add `.` here, let the user deal with this. Along with `NamespacePrefix` property name, this becomes a more general feature - you can have any prefix, with or without `.` at the end.

Reply:
`JavaDomain` is only added when `NamespaceToLower` is true - is that intentional?

Reply:
> I don't think we need to add . here

OK. Fixed.